utils/phpstan-baseline/method.childReturnType.neon

```diff
@@ -137,13 +137,18 @@ parameters:
             count: 1
             path: ../../system/Images/Handlers/BaseHandler.php
 
+        -
+            message: '#^Return type \(CodeIgniter\\Images\\Handlers\\ImageMagickHandler\) of method CodeIgniter\\Images\\Handlers\\ImageMagickHandler\:\:_crop\(\) should be covariant with return type \(\$this\(CodeIgniter\\Images\\Handlers\\BaseHandler\)\) of method CodeIgniter\\Images\\Handlers\\BaseHandler\:\:_crop\(\)$#'
```

Comment:
To fix this, just use `@return $this` in the `_crop` method.

Reply:
Fixed. Thank you.